.golangci.yml

```diff
@@ -41,6 +40,15 @@ linters:
     - unused
     - whitespace
     - unparam
+    - revive
+
+linters-settings:
+  revive:
+    rules:
+      - name: function-result-limit
+        severity: warning
+        disabled: false
+        arguments: [4]
```

Comment:
just curious, what's the args 4 here?

Reply:
It limits the number of returns to 4. I added a comment to explain this